Tests/web3swiftTests/localTests/PromisesTests.swift

```diff
@@ -124,6 +124,6 @@
 //        let userAddress = EthereumAddress("0xe22b8979739D724343bd002F9f432F5990879901")!
 //        let tokenBalance = try await token.read("balanceOf", parameters: [userAddress] as [AnyObject])!.decodedData()
 //        guard let bal = tokenBalance["0"] as? BigUInt else {return XCTFail()}
-//        print(String(bal))
+//        )
 //    }
 // }
```

Comment:
Maybe we should drop this file.
Worth taking a look at what is useful inside and if we don't already have similar tests that weren't using promises.